client/js/lib/addon/loadAddonComponents.js

```diff
@@ -1,11 +1,15 @@
 import { checkResponse, dpRpc } from '@demos-europe/demosplan-ui'
 
 export default async function loadAddonComponents (hookName) {
-  if (window.dplan.loadedAddons[hookName]) {
-    return []
+  while (window.dplan.loadedAddons[hookName] === 'pending') {
+    await new Promise(resolve => setTimeout(resolve, 100))
```

Comment:
```suggestion
    await new Promise(resolve => setTimeout(resolve, 250))
```
maybe a bit more delay?